<launch>
  <!-- calibration interface -->
  <arg name="accel_cmd" default="/calibration/vehicle/accel_cmd"/>
  <arg name="brake_cmd" default="/calibration/vehicle/brake_cmd"/>
  <arg name="steer_cmd" default="/calibration/vehicle/steer_cmd"/>
  <arg name="accel_status" default="/calibration/vehicle/accel_status"/>
  <arg name="brake_status" default="/calibration/vehicle/brake_status"/>
  <arg name="steer_status" default="/calibration/vehicle/steer_status"/>
  <arg name="input_engage_status" default="/api/autoware/get/engage"/>
  <arg name="output_engage_status" default="/calibration/vehicle/is_engage"/>

  <node pkg="calibration_adapter" exec="pacmod_calibration_adapter" name="calibration_adapter" output="screen">
    <remap from="~/input/actuation_command" to="/vehicle/command/actuation_cmd"/>
    <remap from="~/input/actuation_status" to="/vehicle/status/actuation_status"/>
    <remap from="~/input/is_engage" to="$(var input_engage_status)"/>
    <remap from="~/output/accel_cmd" to="$(var accel_cmd)"/>
    <remap from="~/output/brake_cmd" to="$(var brake_cmd)"/>
    <remap from="~/output/steer_cmd" to="$(var steer_cmd)"/>
    <remap from="~/output/is_engage" to="$(var output_engage_status)"/>
    <remap from="~/output/accel_status" to="$(var accel_status)"/>
    <remap from="~/output/brake_status" to="$(var brake_status)"/>
    <remap from="~/output/steer_status" to="$(var steer_status)"/>
    <remap from="~/input/handle_status" to="/vehicle/status/steering_wheel_status"/>
    <remap from="~/output/handle_status" to="/calibration/vehicle/handle_status"/>
    <remap from="~/input/steering_angle_status" to="/vehicle/status/steering_status"/>
    <remap from="~/output/steering_angle_status" to="/calibration/vehicle/steering_angle_status"/>
  </node>
</launch>
